config/application.yml.default

```diff
@@ -130,6 +130,7 @@ idv_acuant_sdk_version_default: '11.8.2'
 idv_acuant_sdk_version_alternate: '11.8.1'
 idv_acuant_sdk_upgrade_a_b_testing_enabled: false
 idv_acuant_sdk_upgrade_a_b_testing_percent: 50
+idv_getting_started_a_b_testing: '{"default":100,"new":0}'
```

Comment:
a nit here: could we go with something like `welcome` and `getting_started`. I think that might help keep things straight as we transition away from the `default` behavior.